easybuild/easyconfigs/p/pigz/pigz-2.3.4_Makefile.patch

```diff
@@ -0,0 +1,17 @@
+--- Makefile.orig	2017-07-12 22:16:56.931252958 -0500
```

Comment:
@JackPerdue please include a brief description of the patch above this line, together with mentioning the author (e.g. `Jack Perdue (TAMU)`)